docs/harness.md

```diff
@@ -180,3 +180,35 @@ Returns the current state being managed by the harness.
 
 Returns the list of blocks that have accumulated by running the harness
 so far.
+
+
+## Testing sagas
+
+We recommend using
+[`redux-saga-test-plan`](https://github.com/jfairbank/redux-saga-test-plan)
+to test sagas; it makes testing sagas (relatively) easy.
+
+Testing sagas requires a slightly different approach than unit testing.
+Since a saga may comprise logic that span multiple asynchronous
+requests, it is often more effective to test that desired effects were
+achieved rather than testing that each step operates correctly.
+
+If you do choose to use a unit testing strategy to test sagas, see
+[Testing Redux
+Sagas](https://hackernoon.com/testing-redux-sagas-c64dbba05935) which
+describes an approach for thoroughly exercising a saga's generator
```

Comment:
That post is reproducing (plagiarizing) this one:
[Conceptualizing Unit Tests with Redux Sagas](https://medium.com/@johnpipkin_79120/conceptualizing-unit-tests-with-redux-sagas-71c49fd8b08f)